pb-rs/src/parser.rs

```diff
@@ -823,8 +985,10 @@ mod test {
         }
     }
 
-    #[test]
-    fn test_missing_tokens() {
+    mod missing_tokens {
```

Comment:
I changed this from a function to an inner-module... This seems like a much better idea than having functions. This way, there is the organization, without the foot-gun of not calling functions!